src/react-16/index.js

```diff
@@ -18,6 +18,11 @@ function react16Selector (selector, parents = rootEls) {
     }
 
     function getName (component) {
+        //react memo
+        // it will find the displayName on the elementType if you set it
+        if (component.elementType && component.elementType.displayName) return component.elementType.displayName;
```

Comment:
First of all - thank you for using TestCafe and contributing to its ecosystem 😄 

Since my understanding of React is not deep enough, may I ask you a question?

Looks like we already have something that is very similar to the added code:
https://github.com/DevExpress/testcafe-react-selectors/blob/650054b096836fc0088cfa08678c2a55fada2ef4/src/react-16/index.js#L33

Could you please clarify the difference between `component.type` and `component.elementType` or point me to corresponding docs or React sources?

Reply:
React memo doesn't return the original component, with its original type (inherited by the ClassName or the Function name), but a memoized one, with the `memo` type

So the check on line `33` will always fail on Memoized component.

I can't find a doc, I arrived to that solution because I knew that using display name was the only way to see the correct name in the Dev Tools.